src/main/java/seedu/eduke8/Eduke8.java

```diff
@@ -63,7 +66,7 @@ private void runCommandLoopUntilExit() {
                 command.execute(topicList, ui);
                 isExit = command.isExit();
             } catch (Eduke8Exception e) {
-                ui.printError();        // able to call ui.printError(e.getMessage()) too if implemented
+                new IncorrectCommand(e.getMessage());
```

Comment:
I think these should not be returning new IncorrectCommands but should be using the ui to print the error instead. the IncorrectCommand will not be executed too.